examples/eos-transactions.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# This is a simple example of a `read-process-write` application
+# using Kafka's transaction API.
+#
+# See the following blog for additional information.
+# https://www.confluent.io/blog/transactions-apache-kafka/
+#
+
+
+"""
+The following example demonstrates how to perform a consume-transform-produce loop with exactly-once semantics.
+
+In order to achieve exactly-once semantics we use the idempotent producer with transactions enabled and a single
```

Comment:
flake8 is 120, do we want to conform to c stlying in python files?

Reply:
I thought it was 80.
```
  --max-line-length=n   Maximum allowed line length for the entirety of this
                        run. (Default: 79)
```

Anyway, it is good practice to keep things below 80, it makes it readable everywhere (terminals, github code reviews, etc), and this is very easy to do with comments that can be formatted in any way we like.
For code it is sometimes not practical to keep below 80, so that's another story.

Reply:
https://github.com/confluentinc/confluent-kafka-python/blob/master/tox.ini#L58

Looks like 3 years ago we configured it to be 119. I'll update the tox.ini as well.